versions/app/src/styles/styles.js

```diff
@@ -183,7 +188,7 @@ const globalStyles =  StyleSheet.create({
     },
     navigate_away_content:{
         flex:1,
-        fontWeight: 'bold',
+        fontWeight: 'medium',
```

Comment:
This throws an error:
> Invalid prop `fontWeight` of value `medium` supplied to `Text`, expected one of ["normal","bold","100","200","300","400","500","600","700","800","900"].

See which one does what you want it to.

Reply:
Thank you for catching this!